src/index.js

```diff
@@ -82,7 +82,7 @@ const addRoutes = (server, stylesheetsPath) => {
     },
     {
       method: 'GET',
-      path: '/eligibility-checker/assets/{file*}',
+      path: '/assets/{file*}',
```

Comment:
Does this work? It shouldn't. Can you test this while using the reverse proxy locally?